todo_app/trello_items.py

```diff
@@ -16,102 +18,39 @@ def Get_trello_APITODOID():
 def Get_trello_APIDOING():
```

Comment:
You have a handful of Trello environment variables listed at the top that I don't think are relevant any longer; it would be good to get rid of those functions if that's correct (and if it's not, we probably want to get rid of them anyway plus whatever's using them!)